src/docs/src/documentation/content/xdocs/zookeeperAdmin.xml

```diff
@@ -1155,6 +1155,30 @@ server.3=zoo3:2888:3888</programlisting>
             </listitem>
           </varlistentry>
 
+          <varlistentry>
+            <term>4lw.commands.whitelist</term>
+
+            <listitem>
+              <para>(Java system property: <emphasis
```

Comment:
Yes, it is a nice trick - if a configuration option 'foobar' in zoo.cfg does not require explicitly handling, then it will implicitly being added as 'zookeeper.foobar' system property during parsing zoo.cfg, see [here](https://github.com/apache/zookeeper/blob/master/src/java/main/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java#L296).